website/projects/awssync_checks.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+import logging
+
+from projects.awssync_structs import AWSTree
+
+
+class Checks:
+    """Class for pipline checks."""
+
+    def __init__(self) -> None:
+        """Initialize Checks class."""
+        self.logger = logging.getLogger("django.aws")
+        self.logger.setLevel(logging.DEBUG)
+
+    def check_members_in_correct_iteration(self, AWSdata: AWSTree) -> None:
+        """Check if the data from the member tag matches the semester OU it is in."""
+        incorrect_emails = []
```

Comment:
I think this array should be called different, in this test we check if the semester tag of the member matches with the iteration it is in. This naming suggest that there are incorrect emails. Maybe use like inconsistent_tags as name.